docs/development/upgrading-chrome.md

```diff
@@ -1,47 +1,65 @@
-# Upgrading Chrome Checklist
+# Upgrading Chromium Workflow
 
 This document is meant to serve as an overview of what steps are needed
-on each Chrome upgrade in Electron.
+on each Chromium upgrade in Electron.
 
-These are things to do in addition to updating the Electron code for any
-Chrome/Node API changes.
+## Update libchromiumcontent (a.k.a. libcc)
+
+- Clone the repo:
+```
+git clone git@github.com:electron/libchromiumcontent.git
+cd libchromiumcontent
+```
+- Run bootstrap script to init and sync git submodules:
+```
+./script/bootstrap -v
+```
+- Update `VERSION` file to correspond to Chromium version.
+- Run `script/update`, it will probably fail applying patches.
+- Fix failing patches. `script/patch.py` might help.
+  - Don't forget to fix patches in the `patches-mas/` folder.
+- Build libcc:
+```
+./script/build
+```
+- Create dist folders which will be used by electron:
+```
+./script/create-dist --no_zip
+cd dist/main
+../../tools/generate_filenames_gypi filenames.gypi src shared_library static_library
+cd -
+```
+- Open a pull request to `electron/libchromiumcontent` with the changes.
+- Fix compilation on the all supported platforms/arches.
+
+## Update Electron
+
+- Set `vendor/libchromiumcontent` revision to a version with the new Chromium.
+  - It will be great if GH builds for this libcc version are already green
+    and its archives are already available. Otherwise everyone would need
+    to build libcc locally in order to try build a new Electron.
+- Set `CLANG_REVISION` in `script/update-clang.sh` to match the version
+  Chromium is using. You can find it in file `src/tools/clang/scripts/update.py` in updated `electron/libchromiumcontent` repo.
+- Run `script/bootstrap.py`.
```

Comment:
Maybe we should mention `--libcc_source_path=` and other `--libcc_...` arguments of `bootstrap` script?
It might be easier to use them than to create a symlink, especially on Windows.